THIRD-PARTY-LICENSES.txt

```diff
@@ -339,12 +339,12 @@ Lists of 360 third-party dependencies.
      (Apache License 2.0) swagger-annotations (io.swagger:swagger-annotations:1.6.0 - https://github.com/swagger-api/swagger-core/modules/swagger-annotations)
      (Apache License 2.0) swagger-core (io.swagger:swagger-core:1.6.0 - https://github.com/swagger-api/swagger-core/modules/swagger-core)
      (Apache License 2.0) swagger-integration (io.swagger.core.v3:swagger-integration:2.1.1 - https://github.com/swagger-api/swagger-core/modules/swagger-integration)
-     (Apache Software License, Version 2.0) swagger-java-bitbucket-client (io.dockstore:swagger-java-bitbucket-client:1.10.0-alpha.0-SNAPSHOT - https://github.com/dockstore/dockstore/swagger-java-bitbucket-client)
```

Comment:
Ideally, in that ticket we also ignore/skip our own project (not third party) in this file.

Reply:
Done. Included that in the ticket that was just created.